### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by gavinr**
Adding some notes about the release process to make it smoother going forward.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- Check out the repository and pull the latest updates on MASTER (origin or your fork should work)
+- Update the package version in `package.json`.
+- Update `CHANGELOG.md` - remember to update the links at the bottom of the document.
+- Run `npm install`
```
Comment (jgravois) [inline RELEASE.md]: did you mean `npm version`?

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Commit with message `:package: X.X.X` (replacing with version)
+- Push commit to master
+- Run `npm whoami`, making sure you're logged in properly to NPM. If not, `npm login`.
+- Run `scripts/release.sh`
```
Comment (jgravois) [inline RELEASE.md]: nothing wrong with what you have, but `npm run release` is handy too.

Comment (gavinr) [reply RELEASE.md]: Good point. Updated. Thanks @jgravois

File: RELEASE.md (hunk 1 above)
Comment (jgravois) [reply RELEASE.md]: > One time I did `npm version` and then later `gh-release` failed

now that I think about it, that actually makes sense. `gh-release` helps with creating a tag (and release) on GitHub using the information in the CHANGELOG. it expects that you will have bumped the version number in your package.json file manually, but _not_ already tagged a version.

`npm version` bumps your package.json and creates a (local) tag.

**Pull request merged**